template/web/gatsby-config.js

```diff
@@ -17,8 +18,8 @@ module.exports = {
       options: {
         ...clientConfig.sanity,
         token: process.env.SANITY_READ_TOKEN,
-        watchMode: !isProd,
-        overlayDrafts: !isProd,
+        watchMode: !isPreview,
```

Comment:
@pvorozhe with this value for `watchMode`, it'll be used in builds, which could lead to some issues... Could you change this to `!isProd && !previewEnabled`? Thanks 🙇‍♂️

Reply:
@hdoro i am a bit confused on this part. what do you mean by previewEnabled?

Reply:
@pvorozhe sorry for the wait!

Sorry for the typo in my initial message, what I mean is `!isProd && !isPreview` :)